corehq/apps/export/models/new.py

```diff
@@ -148,13 +148,15 @@ def create_default_from_export_item(group_schema_path, item, app_ids_and_version
         """
 
         is_main_table = group_schema_path == MAIN_TABLE
+        is_advanced = isinstance(item, SystemExportItem) and item.is_advanced
```

Comment:
Oh I like that idea of creating a `SystemColumn`. I'll look into how hard/easy that might be